fidesctl/src/fidesapi/main.py

```diff
@@ -73,6 +74,18 @@ async def health() -> Dict:
     return {"data": {"message": "Fidesctl API service is healthy!"}}
 
 
+@app.get("/version", tags=["Version"])
+async def version() -> Dict:
+    "Return the version of the fidesctl app that is running."
+    server_version = str(fidesctl.__version__)
+    return {
+        "data": {
+            "message": f"Fidesctl version: {server_version}",
+            "version": server_version,
+        }
+    }
```

Comment:
Instead of adding a new endpoint, how do you feel about adding a `version` key to the `/health` response instead? Not a big deal either way, but I think it makes sense and would prevent the need for multiple API calls elsewhere.

If you want to keep the dedicated endpoint, I think we need should somehow specify in the docstring and the `message` that we're returning the API server version.

Reply:
ohhhh good call here. I like adding it to the `health` endpoint.